src/test/groovy/com/google/protobuf/gradle/ProtobufAndroidPluginKotlinTest.groovy

```diff
@@ -8,9 +8,9 @@ import spock.lang.Unroll
 
 @CompileDynamic
 class ProtobufAndroidPluginKotlinTest extends Specification {
-  private static final List<String> GRADLE_VERSION = ["5.6", "6.5.1", "7.4.2"]
-  private static final List<String> ANDROID_PLUGIN_VERSION = ["3.5.0", "4.1.0", "7.2.1"]
-  private static final List<String> KOTLIN_VERSION = ["1.3.20", "1.3.20", "1.3.40"]
+  private static final List<String> GRADLE_VERSION = ["5.6", "6.5.1", "7.4.2", "7.6"]
```

Comment:
We can. I'd prefer to check more agp and kotlin versions. In test we use unroll with that `where`, because of this length of the arrays should be same.
```
   where:
    agpVersion << ANDROID_PLUGIN_VERSION
    gradleVersion << GRADLE_VERSION
    kotlinVersion << KOTLIN_VERSION
```